### PR TITLE
Remove auto-focus on certain input fields

### PR DESCRIPTION
**Pull request by lordelix**
## Description
Remove auto-focus on certain input fields if browser is part of the MSIE family.

## Related Issue
https://github.com/owncloud/core/issues/22689

## Motivation and Context
Focus on input fields immediately removes the placeholder value, thus concealing the input label.

## How Has This Been Tested?
Tested using **MSIE11** and **MS Edge** on Browserstack.com

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
don't add node_modules - thx

**Comment by felixheidecke** [comment]
> don't add node_modules - thx

ups. Thought they were part of .gitignore :fearful:

**Comment by DeepDiver1975** [comment]
> ups. Thought they were part of .gitignore :fearful:

it is - but if you do a git add  node_modules they  go in :wink:

**Comment by michaelstingl** [comment]
I checked input fields of the Sharing panel. Autofocus is disabled in IE11. 👍 

I'm not aware of issues with MS Edge Browser, so disabling it for Edge shouldn't be necessary.

**Review by michaelstingl — CHANGES_REQUESTED** [review]
I checked input fields of the Sharing panel. Autofocus is disabled in IE11. 👍

I'm not aware of issues with MS Edge Browser, so disabling it for Edge shouldn't be necessary.

File: core/js/sharedialoglinkshareview.js [hunk 1]
```diff
@@ -193,7 +193,9 @@
 					password: ''
 				});
 			} else {
-				this.$el.find('.linkPassText').focus();
+				if (!bowser.msie) {
```
Comment (PVince81) [inline core/js/sharedialoglinkshareview.js]: We already had a method `OC.Util.isIE()` for this.
Sorry I didn't notice earlier...

Comment (felixheidecke) [inline core/js/sharedialoglinkshareview.js]: This should be replaced by bowser.js in the future to test against version, os and other devices. Could even test before the DOM is ready. Better after 10.0 release tho. :wink:
Should I revert or keep it in for future use?

Comment (PVince81) [inline core/js/sharedialoglinkshareview.js]: Keep it, but maybe make OC.isIE also use that?

Comment (felixheidecke) [inline core/js/sharedialoglinkshareview.js]: I'm on it ;-)

**Comment by felixheidecke** [comment]
@michaelstingl please re-evaluate.

**Review by michaelstingl — APPROVED** [review]

**Pull request merged**